test/bin/get_latest_rpm_version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#
+# This script reports the version of the latest RPM in the local
+# repository.
+
+set -euo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source "${SCRIPTDIR}/common.sh"
+
+release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm')
```

Comment:
Is there a chance we'll get more than one RPM? If so, we should use `sort | tail -1` on the result of `find`

Reply:
The framework automation prevents it, but I'll push a fix anyway after the tests finish running with this version.